std_msgs/Header header
bool isFixed
geometry_msgs/Pose pose
sensor_msgs/Image leftImage
sensor_msgs/Image rightImage